docs/_snippets/webhook-metadata-post-example.mdx

```diff
@@ -0,0 +1,8 @@
+```json
+{
+  "metadata": {
+     youCan: "putAny",
+     stringKeyValue: "pairsHere",
+  }
+}
```

Comment:
why the overhead of `metadata.metadata`, can we not just have metadata be the key-value pair dictionary?

ie suggest unwrapping metadata to just an object unless there's good reason otherwise

Reply:
I agree we don't need `metadata.metadata`. Just to be clear, I think this should be the payload/body to the API:

```json
{
  "metadata": {
    "someprop": "with whatever value"
  }
}
```

when using the SDK, I think this should be the experience:

```ts
const myPayload: Record<string, string> = { someprop: "with whatever value" };
await medicalApi.startDocumentQuery(patientId, facilityId, myPayload);
// ^ the SDK deals with wrapping the 3rd param into the `metadata` request body prop.
```